extension.Rmd

```diff
@@ -0,0 +1,175 @@
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
+dev.off()
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred, actual = actual, type = "rect", heading = "Model2: The Parsons Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 2](figures/SP.model2.png)
+
+#Conclusion:
+Again, there is little variation in our predictive probabilities with this model. Similar to model 1, most events are aligned with higher predictive probabilities on the right hand of the graph. Yet Model 1 has slightly better predictive power as the very end of the graph is likely to be clustered with event than non-events as in Model 2. 
+
+```
+
+# Model 3: Group Conflict Model (Various theories)
+
+Group conflict theories encompass a variety of theories.  Some emphasize the role of instrumental rationality, arguing that xenophobia results from a contest between immigrants and residents over scarce resources like jobs and welfare benefits. Others emphasize the scapegoating of immigrants as perceived threats to residents' status, power or resources.  In this last model, we use relevant variables representing both the availability of resources like jobs and welfare benefits and the prevalence of immigration: unemployment rates, unemployment benefits, number of asylum seekers, saliance and variance of right-wing issues.
+
+```{r}
+model_3 <- glm(ext_vote ~ z_asylumseeker + z_unemp + z_asylumseeker:z_unemp + z_replacement + z_replacement:z_unemp + z_replacement:z_asylumseeker + country - 1, data = data)
+
+summary(model_3)
+actual <- model_3$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_3$fitted.values # Create a function with log-odds
+# Create link function to transform log-odds into predicted probabilities
+link <- function(P) {
+  1/(1+exp(-P))
+} 
+pred <- link(log.odds) # Transform log-odds into predicted probabilities
+pred <- as.vector(pred) 
+range(pred) # variation in the predicted probabilities: very little variation.
+dev.off()
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred, actual = actual, type = "rect", heading = "Model3: The Group Conflict Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 3](figures/SP.model3.png)
+
+#Conclusion:
+The majority of events (intention to vote for ER) are clustered at the end right end of the model - which is in line with a good predictive model. However, in this model the distribution of events is very dispersed compared to other two models.
+
+All Three Models
+
+![Model 1](figures/SP.Model1.png)
+
+![Model 2](figures/SP.model2.png)
+
+![Model 3](figures/SP.model3.png)
+
```

Comment:
my impression was that your extension project was more clear than your replication project. it seems that this was due in part because you had more control over what you wanted to do, and the tools you were using to do it. i really liked the use of separation plots.